demos/src/css/chartAuthoring-demo.css

```diff
@@ -1,33 +1,183 @@
-.floec-ca-pieChart-pie {
+.main {
+    width: 100%;
+    font-family: Verdana, Arial, sans-serif;
+}
+
+/* Main container */
```

Comment:
The class name used at line 1 is called ```main``` that seems more applicable to use "main container". You might wanna be more specific if you meant to explain the class at line 8.